x-pack/plugins/alerting/server/config.ts

```diff
@@ -60,25 +60,24 @@ const rulesSchema = schema.object({
   }),
 });
 
-export const DEFAULT_MAX_EPHEMERAL_ACTIONS_PER_ALERT = 10;
-export const configSchema = schema.object({
-  healthCheck: schema.object({
-    interval: schema.string({ validate: validateDurationSchema, defaultValue: '60m' }),
-  }),
-  invalidateApiKeysTask: schema.object({
-    interval: schema.string({ validate: validateDurationSchema, defaultValue: '5m' }),
-    removalDelay: schema.string({ validate: validateDurationSchema, defaultValue: '1h' }),
-  }),
-  maxEphemeralActionsPerAlert: schema.number({
-    defaultValue: DEFAULT_MAX_EPHEMERAL_ACTIONS_PER_ALERT,
-  }),
-  enableFrameworkAlerts: schema.boolean({ defaultValue: true }),
-  cancelAlertsOnRuleTimeout: schema.boolean({ defaultValue: true }),
-  rules: rulesSchema,
-  rulesSettings: schema.object({
-    cacheInterval: schema.number({ defaultValue: DEFAULT_CACHE_INTERVAL_MS }),
-  }),
-});
+export const configSchema = schema.object(
+  {
+    healthCheck: schema.object({
+      interval: schema.string({ validate: validateDurationSchema, defaultValue: '60m' }),
+    }),
+    invalidateApiKeysTask: schema.object({
+      interval: schema.string({ validate: validateDurationSchema, defaultValue: '5m' }),
+      removalDelay: schema.string({ validate: validateDurationSchema, defaultValue: '1h' }),
+    }),
+    enableFrameworkAlerts: schema.boolean({ defaultValue: true }),
+    cancelAlertsOnRuleTimeout: schema.boolean({ defaultValue: true }),
+    rules: rulesSchema,
+    rulesSettings: schema.object({
+      cacheInterval: schema.number({ defaultValue: DEFAULT_CACHE_INTERVAL_MS }),
+    }),
+  },
+  { unknowns: 'allow' }
```

Comment:
**nit:** Preventing unknowns adds the benefit of users knowing when they have typos and such. Perhaps if we change `maxEphemeralActionsPerAlert` to a `schema.maybe(schema.number())`, we won't see it anywhere else (it'll be empty by default now)